### PR TITLE
Use consistent stylesheet indentation

### PR DESCRIPTION
**Pull request by kaishin**
### Motivation:

Stylesheet source files used a mixture of 2-space and 4-space based indentation.

### Modifications:

Make all stylesheet files use 2-space indentation.

### CODE REVIEW

**Review by alexandersandberg — COMMENTED** [review]
I don't think this is worth doing unless we add Prettier or some other automatic formatting. Otherwise, things will get messy right away again.

Also, which config to use is quite opinionated. Maybe the default Prettier config is the one to go with, as it should be the least opinionated.

**Comment by kaishin** [comment]
Update: this came up during the workgroup meeting. The agreement is to add Prettier as part of this PR and tackle automating the checks in a separate task. Also merging should wait for #344 to be merged first.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "overrides": [
+    {
+      "files": "**/*.scss",
```
Comment (alexandersandberg) [inline .prettierrc:4]: We should also format `.css` files. There are 2 in the project at the moment.

```suggestion
      "files": ["**/*.css", "**/*.scss"],
```

**Review by daveverwer — APPROVED** [review]

**Review by 0xTim — APPROVED** [review]
LGTM once the merge conflicts are fixed

**Comment by kaishin** [comment]
@swift-ci please test

**Comment by kaishin** [comment]
@alexandersandberg I went with the absolute minimum Prettier config and re-run it on all files again.

**Review by daveverwer — APPROVED** [review]
👍

**Pull request merged**